src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -10,7 +10,8 @@ import _root_.slick.driver.JdbcProfile
 import com.typesafe.config.{Config, ConfigFactory, ConfigValueFactory}
 import cromwell.engine.ExecutionIndex._
 import cromwell.engine.ExecutionStatus._
-import cromwell.engine.backend.{Backend, JobKey, WorkflowQueryResult}
+import cromwell.engine.backend._
+import cromwell.engine.backend.jes.JesBackend
```

Comment:
Ultimately that's true of course, but I wouldn't worry about this too much now if it causes you pain.  There are already explicit references to the three backend implementations here.  :unamused: